clang/test/3C/canwrite_constraints.h

```diff
@@ -19,3 +19,9 @@ int *foo_var = ((void *)0);
 // dedicated test for it.
 inline void no_op() {}
 // CHECK_HIGHER: inline void no_op() _Checked {}
+
+// In the lower case, this should stay wild
```

Comment:
Please add an `expected-warning` for the root cause warning for the typedef, like the others above.